e2e/node/manager_test.go

```diff
@@ -446,8 +446,7 @@ func (s *ManagerTestSuite) TestRaceConditions() {
 	}
 
 	time.Sleep(2 * time.Second)                // so that we see some logs
-	nodeStopped, e := s.NodeManager.StopNode() // just in case we have a node running
-	s.NoError(e)
+	nodeStopped, _ := s.NodeManager.StopNode() // just in case we have a node running
```

Comment:
We shouldn't assert stopping node here because the node may be already stopped: the order is random by design of this test.